x-pack/plugins/cases/public/components/case_view/case_view_page.tsx

```diff
@@ -196,14 +122,25 @@ export const CaseViewPage = React.memo<CaseViewPageProps>(
             </EuiFlexItem>
           </EuiFlexGroup>
 
-          <EuiSpacer size="xs" />
-
-          <EuiTabs>{renderTabs()}</EuiTabs>
-
           <EuiSpacer size="l" />
 
-          <EuiFlexGroup data-test-subj={`case-view-tab-content-${activeTabId}`}>
-            {selectedTabContent}
+          <EuiFlexGroup
+            data-test-subj={`case-view-tab-content-${activeTabId}`}
+            alignItems="baseline"
+          >
+            {activeTabId === CASE_VIEW_PAGE_TABS.ACTIVITY && (
+              <CaseViewActivity
+                ruleDetailsNavigation={ruleDetailsNavigation}
+                caseData={caseData}
+                actionsNavigation={actionsNavigation}
+                showAlertDetails={showAlertDetails}
+                useFetchAlertData={useFetchAlertData}
+                activeTab={activeTabId}
```

Comment:
I don't think we need the `activeTab` prop. The component `CaseViewActivity` can pass `CASE_VIEW_PAGE_TABS.ACTIVITY` to the `CaseViewTabs` component. Same for `CaseViewAlerts`.